src/app.py

```diff
@@ -84,8 +93,21 @@ def on_identity_loaded(sender, identity):
 
     login_manager.init_app(app)
 
+    app.logger.info('Bootstrapped app')
+
     return app
 
 if __name__ == '__main__':
     app = create_app()
-    app.run()
\ No newline at end of file
+    app.run()
+
+def __init_logging(app):
+    rootLogger = logging.getLogger()
+    rootLogger.setLevel(logging.INFO)
+    rootLogger.addHandler(default_handler)
+
+    if os.environ['LOGGLY_TOKEN'] is not None:
+        handler = HTTPSHandler(f'https://logs-01.loggly.com/inputs/{os.environ["LOGGLY_TOKEN"]}/tag/todo-app')
+        handler.setFormatter(Formatter("[%(asctime)s] %(levelname)s in %(module)s: %(message)s"))
+        app.logger.addHandler(handler)
+        rootLogger.addHandler(handler)
```

Comment:
I don't think you need to add the handler to app.logger if you've added it to the root logger. You'll get duplicate logs unless you set `app.logger.propagate = False`